tests/observability/metrics/test_metrics.py

```diff
@@ -175,6 +179,67 @@ def test_kubevirt_vmi_info(self, prometheus, single_metric_vm, single_metric_vmi
             values_to_compare=single_metric_vmi_guest_os_kernel_release_info,
         )
 
+    @pytest.mark.polarion("CNV-11862")
+    def test_metric_kubevirt_vm_info(self, prometheus, single_metric_vm, vm_info_to_compare):
+        compare_kubevirt_vmi_info_metric_with_vm_info(
+            prometheus=prometheus,
+            query=KUBEVIRT_VM_INFO.format(vm_name=single_metric_vm.name),
+            expected_value="1",
+            values_to_compare=vm_info_to_compare,
+        )
+
+    """Windows vms"""
+
+    @pytest.mark.polarion("CNV-11859")
+    def test_vmi_domain_total_memory_bytes_windows(
+        self,
+        windows_vm_for_test,
+        vmi_domain_total_memory_in_bytes_from_windows_vm,
+        windows_vmi_domain_total_memory_bytes_metric_value_from_prometheus,
+    ):
+        """This test will check the domain total memory of VMI with given metrics output in bytes."""
+        assert (
+            vmi_domain_total_memory_in_bytes_from_windows_vm
+            == windows_vmi_domain_total_memory_bytes_metric_value_from_prometheus
+        ), (
+            f"VM {windows_vm_for_test.name}'s domain memory total {vmi_domain_total_memory_in_bytes_from_windows_vm} "
+            f"is not matching with metrics value "
+            f"{windows_vmi_domain_total_memory_bytes_metric_value_from_prometheus} bytes."
+        )
+
+    @pytest.mark.polarion("CNV-11860")
+    def test_vmi_used_memory_bytes_windows(
+        self,
+        prometheus,
+        windows_vm_for_test,
+        updated_dommemstat_windows,
+    ):
+        if is_jira_open(jira_id="CNV-59552"):
+            pytest.xfail(
+                reason="Expected failure, "
+                "there is a bug with this metric on windows vms because of virsh dommemstat command."
```

Comment:
```suggestion
                reason="Expected failure, "
                "there is a bug (CNV-59552) with this metric on windows vms because of virsh dommemstat command."
```